db/schema.rb

```diff
@@ -178,6 +182,17 @@
     t.integer "grade"
   end
 
+  create_table "course_assessment_question_text_responses", force: :cascade do |t|
+  end
+
+  create_table "course_assessment_question_text_response_solutions", force: :cascade do |t|
+    t.integer "question_id",   null: false, index: {name: "fk__course_assessment_text_response_solution_question"}, foreign_key: {references: "course_assessment_question_text_responses", name: "fk_course_assessment_questi_2fbeabfad04f21c2d05c8b2d9100d1c4", on_update: :no_action, on_delete: :no_action}
```

Comment:
I was following this .. https://github.com/Coursemology/coursemology2/blob/master/db/migrate/20150726062900_create_course_assessment_question_multiple_response.rb